app/javascript/.storybook/main.js

```diff
@@ -86,6 +86,25 @@ module.exports = {
   },
   babel: async (options) => ({
     ...options,
-    plugins: [...options.plugins, 'inline-react-svg'],
+    plugins: [
```

Comment:
I'm going to create a separate PR for the storybook and babel config changes as they affect everything at the moment and it's easier to track the change in git history.